### PR TITLE
Misc cleanup

### LINKED ISSUE #61: Specify the Bypass Transaction

### LINKED ISSUE #66: Rename `id` to `txid` in messages
When we are really designing the `txid` of the transaction, use that. Opt for `id` when designing the `outpoint` that can be seen as a "vault id"

### LINKED ISSUE #67: Move get_spend_tx and spent_tx to the Synchronisation Server section

### LINKED ISSUE #69: Move all communication channels to KK and rename network.md to communications.md (or something else more specific)

### PR DESCRIPTION
**Pull request by JSwambo**
Fixes #69
Fixes #67
Fixes #66
Fixes #61

### CODE REVIEW

File: transactions.md [hunk 1]
```diff
@@ -104,15 +105,15 @@ The CPFP output value is adjusted depending on the actual transaction size.
 - outputs[1]:
     - value: N/A (may be many of them)
     - scriptPubkey: N/A
-- outputs[N]:
+- outputs[n]:
```
Comment (darosior) [inline transactions.md:108]: Why ? :)

File: transactions.md [hunk 2]
```diff
@@ -145,7 +146,7 @@ transactions are never meant to be used.
 
 ### vault_emergency_tx
 
-The transaction which spends the [`vault_tx`](vault_tx) output to the EDV by the `N`-of-`N` path.
+The transaction which spends the [`deposit_tx`](deposit_tx) output to the EDV by the `N`-of-`N` path.
```
Comment (darosior) [inline transactions.md:149]: Nice catch

File: transactions.md [hunk 3]
```diff
@@ -191,3 +192,38 @@ This transaction spends the [`unvault_tx`](unvault_tx) `output[0]` to the EDV by
...
+
+
+## bypass_tx
+
+Bypass txs are used in the case where stakeholders need immediate access to funds. Bypass 
+txs avoid the controls on expenses set by stakeholders and enforced by watchtowers. 
+
+A Bypass transaction spends the [`deposit_tx`](deposit_tx) output by the `N`-of-`N` path and 
+pays to a arbitrary addresses.
+
```
Comment (darosior) [inline transactions.md:204]: > by the `N`-of-`N` path

Well that's the only one :) Can you remove this part ?

File: transactions.md [hunk 4]
```diff
@@ -191,3 +192,38 @@ This transaction spends the [`unvault_tx`](unvault_tx) `output[0]` to the EDV by
...
+#### IN
+
+- count: 1
+- inputs[0]:
+    - txid: `<deposit_tx txid>`
+    - vout: `<deposit_tx vout>`
+    - sequence: `0xfffffffd`
+    - scriptSig: `<empty>`
+    - witness: `satisfy(vault_descriptor)`
```
Comment (darosior) [inline transactions.md:216]: Arg we should rename this descriptor to `deposit` too (not necessarily in this PR)

File: transactions.md [hunk 5]
```diff
@@ -191,3 +192,38 @@ This transaction spends the [`unvault_tx`](unvault_tx) `output[0]` to the EDV by
...
+
+#### OUT
+
+- count: 1
```
Comment (darosior) [inline transactions.md:220]: Count is unspecified. Actually we should just put "Unspecified" for the `#### OUT` section, i think

**Review by darosior — COMMENTED** [review]
Nice, just a few comments

File: transactions.md (hunk 1 above)
Comment (JSwambo) [reply transactions.md:108]: A reader may be confused by the other usage of N for the N stakeholders.

File: transactions.md [hunk 6]
```diff
@@ -191,3 +192,27 @@ This transaction spends the [`unvault_tx`](unvault_tx) `output[0]` to the EDV by
 - outputs[0]:
     - value: `<unvault_tx outputs[0] value - fees>`
     - scriptPubkey: `0x00 SHA256(<EDV_script>)`
+
+
+## bypass_tx
+
+Bypass txs are used in the case where stakeholders need immediate access to funds. Bypass 
+txs avoid the controls on expenses set by stakeholders and enforced by watchtowers. A 
+Bypass tx spends the [`deposit_tx`](deposit_tx) and pays to a arbitrary addresses.
```
Comment (darosior) [inline transactions.md:201]: nit:
> to *an arbitrary address*

or even preferably

> to *arbitrary addresses*

**Review by darosior — COMMENTED** [review]
Just a final small typo

**Review by darosior — APPROVED** [review]
ACK 3f0b25d5bc823580b376c332d37c2f72ee869fb0

**Comment by darosior** [comment]
Actually this did not rename network.md .. https://github.com/re-vault/practical-revault/issues/69

**Comment by darosior** [comment]
Fixed in https://github.com/re-vault/practical-revault/pull/76

**Pull request merged**